backend/app/services/integration_service.py

```
@@ -0,0 +1,230 @@
+                .select("*")\
+                .eq("id", str(integration_id))\
+                .eq("user_id", str(user_id))\
+                .execute()
+
+            if not result.data:
+                return None
+
+            return IntegrationResponse(**result.data[0])
+
+        except Exception as e:
+            logger.error(f"Error getting integration: {str(e)}")
```

Ruff (0.14.0)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)